front-end/src/components/countries-search-page/CountrySearchPage.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable max-len */
+import * as React from 'react';
+// import Stack from '@mui/material/Stack';
```

Comment:
Lots of commented out code -- should this be removed? 
- import statements
- lines 45-68